test/commands/new.test.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from '@oclif/test';
+
+describe('new', () => {
+  test
+    .stderr()
+    .stdout()
+    .command(['new', '--no-tty', '-n=specification.yaml'])
+    .it('runs new command', ctx => {
+      expect(ctx.stdout).to.contain('specification.yaml');
+    });
```

Comment:
```suggestion
    .it('runs new command', (ctx, done)=> {
      expect(ctx.stdout).to.contain('specification.yaml');
      done();
    });
```

can you do this, I think the test is not waiting for the command to finish.

Reply:
Still, getting errors 😅😅

Reply:
![image](https://user-images.githubusercontent.com/54466041/153897168-ee8d045f-3238-44c1-a2b8-a9be8bca4a01.png)

Reply:
If you see the logs for the second test you can see the output from the new command is being used for assertion and that is failing all the other tests.

Reply:
Yeah, that's in the validation, I guess it's clashing with the new's test.
I am unable to figure out how to fix it.
Please help me out 🙂